go.mod

```diff
@@ -1,31 +1,31 @@
 module github.com/replicatedhq/kots
 
-go 1.17
+go 1.19
```

Comment:
with this update to 1.19 in the mod file, how are you running `go mod tidy` or equivalent commands?  when I run `go mod tidy` locally I now get the following error: `go: go.mod file indicates go 1.19, but maximum version supported by tidy is 1.18`

Reply:
The commands work for me. I'm using Go 1.19. I was unsure if I needed to do this at all, but it seems like some dependency train started after adding the new troubleshoot that led me here? If I can go back to 1.17 I will, but not sure.

Reply:
ah gotchya, I likely just need to update, thanks